webapp-django/crashstats/settings/base.py

```diff
@@ -715,6 +718,16 @@ def path(*dirs):
             'virtual_host': config('RABBITMQ_VIRTUAL_HOST', '/'),
             'port': config('RABBITMQ_PORT', 5672),
         },
+        'boto': {
+            'access_key': config('resource.boto.access_key', ''),
+            'bucket_name': config(
+                'resource.boto.bucket_name', 'crashstats'),
+            'prefix': config('resource.boto.prefix', ''),
+            'keybuilder_class': config(
+                'resource.boto.keybuilder_class',
+                'socorro.external.boto.connection_context.DatePrefixKeyBuilder'
+            ),
+        }
```

Comment:
That doesn't really make sense, because you'll have to rewrite them to the django namespace anyway. I can see beauty in the consistency between namespaces, but I think we should favor consistency inside the namespace more. Everything in the django namespace is uppercase, we should stick to that. 

But, I'm not going to block merging this PR on that. :)